src/gcp_scanner/crawl.py

```diff
@@ -595,9 +591,9 @@ def get_pubsub_subscriptions(project_id: str,
         project=f"projects/{project_id}")
     while request is not None:
       response = request.execute()
-      for subscription in response.get("subscriptions", []):
-        pubsubs_list.append(subscription)
-
+      pubsubs_list = [subscription 
+                      for subscription in response.get("subscriptions", [])]
+      
```

Comment:
Trim trailing whitespace

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.